javascript-loops-practice/javascript-loops-practice/add-suffix-to-all.js

```diff
@@ -0,0 +1,9 @@
+/* exported addSuffixToAll */
+function addSuffixToAll(words, suffix) {
+  var suffixArray = [];
+
+  for (var i = 0; i < words.length; i++) {
+    suffixArray.push(words[i] += suffix);
```

Comment:
Whoops! By using the `+=` operator here, you are changing the values in the original array, which is a no-no. Try to do this without changing the original array values 👍